src/templates/Brochure/BrochureTemplate.tsx

```diff
@@ -152,32 +145,68 @@ const Page1: React.FC<{ document: BrochureDocument }> = ({ document }) => {
         } else {
           return (
             <>
-              <h5>{section.subheader}</h5>
+              <div className="text-xl font-semibold">{section.subheader}</div>
               <div>{section.body}</div>
               <br />
             </>
           );
         }
       })}
-    </BrochurePage>
+    </>
   );
+
+  if (isMobile) {
+    const header = document.page1.header;
+    const description = document.shared.tradeTrust.description;
```

Comment:
am i seeing the wrong commit here? i didn't see these in previous commits

Reply:
im generally not agreeable with having separate mobile and desktop views, it'll be easier 2 fold maintenance. in future should push back design. for urgency sake, this shall see no evil for now